tests/test_sync_commands.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+import os
+import unittest
+
+import yaml
+from syncano.models import RuntimeChoices
+from syncano_cli.config import ACCOUNT_CONFIG, ACCOUNT_CONFIG_PATH
+from syncano_cli.main import cli
+from tests.base import InstanceMixin, IntegrationTest
+
+
+class BaseSyncCLITest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(BaseSyncCLITest, cls).setUpClass()
+        cls.yml_file = 'syncano.yml'
+        cls.scripts_dir = 'scripts/'
+
+    def _assert_file_exists(self, path):
```

Comment:
Why are all these asserts private (or well, prefixed with '_' as concept of private doesn't exactly exist in python)?
They're completely valid and useful asserts. I think they should be public just like other assert methods on testcase class are.